arch/x86/kernel/slaunch.c

```diff
@@ -27,13 +27,72 @@
 #include <asm/e820/api.h>
 #include <asm/bootparam.h>
 #include <asm/setup.h>
+#include <linux/fs.h>
```

Comment:
This should go in a separate patch to make our rebases of the patch set go as smoothly as they can.

Reply:
Specifically this on line include should go with 2ba13df24ceca280eec1e42a3fbdc60a1f3fc4c8.

Reply:
linux/fs is no longer needed so I just removed this line

Reply:
ack